src/main/java/hellos/Hellos.java

```diff
@@ -35,9 +36,12 @@ private String kkSaysHello() {
     return "KK says 'Hello!'\n";
   }
 
+alice-greeting
+  private String aliceSaysHello(){
```

Comment:
The "standard" is to have a space before the curly brace at the start of a method:

```suggestion
  private String aliceSaysHello() {
```